pkg/asset/installconfig/gcp/validation.go

```diff
@@ -36,6 +37,10 @@ var computeReq = resourceRequirements{
 func Validate(client API, ic *types.InstallConfig) error {
 	allErrs := field.ErrorList{}
 
+	if err := validate.GCPClusterName(ic.ObjectMeta.Name); err != nil {
```

Comment:
If a cluster name violates the RFC-1035 requirements, the installer will report that twice.
https://github.com/openshift/installer/blob/296b0b10da2a5a9a5afca6c10b03ee73ce98578c/pkg/types/validation/installconfig.go#L73

Reply:
Removed the RFC check in the GCPClusterName function.